bin/start_backend.sh

```diff
@@ -177,9 +180,18 @@ if [ ${RUN_CN} -eq 1 ]; then
     LOG_FILE=${LOG_DIR}/cn.out
 fi
 
-echo "start time: "$(date) >> ${LOG_FILE}
+if [ ${RUN_LOGSTDOUT} -eq 1 ] ; then
+    # force glog output to console, redirect stderr to stdout
+    export GLOG_logtostderr=1
+    exec 2>&1
```

Comment:
I think we can let error goes to `2` `stderr` so this let datadog agent to add the default status of `Error` when export to DD backend.

Reply:
done